NetCord/Gateway/GatewayClient.cs

```diff
@@ -8,6 +8,9 @@
 
 namespace NetCord.Gateway;
 
+/// <summary>
+/// The WebSocket-based client allows you to receive events from Discord's Gateway and update or request resources, such as updating voice state.
```

Comment:
```suggestion
/// The WebSocket-based client that allows you to receive events from the Discord's Gateway and update or request resources, such as updating voice state.
```

Reply:
The previous description is better grammatically / in terms of readability.

Reply:
Why? I think the new one is better because it says that it's `the websocket client **that** allows you to ...`.

Reply:
English is weird, but the way you wrote it originally is better / actually the standard.
![image](https://github.com/NetCordDev/NetCord/assets/114494949/d8e41cee-9145-427f-bbe2-6103d5154252)
The latter section of `receive events from the Discord's Gateway` is the incorrect part.

Reply:
For reference, adding a 'that' grammatically implies there's only one GatewayClient, which is untrue, since its an object you instantiate and not a constant part of the runtime or something.

Reply:
Ok, you are correct I think. I have even found a better [example](https://learn.microsoft.com/en-us/dotnet/api/system.net.websockets.websocket?view=net-8.0). Although I don't get why `receive events from the Discord's Gateway ` is incorrect. Discord docs contains for example:
> The Gateway API

> The Gateway is Discord's form of

~ https://discord.com/developers/docs/topics/gateway

Reply:
Yep, those are correct, because prefixing the word 'Discord' before 'Gateway' causes the issue. It instead turns the sentence's semantics into "The Discord's Gateway", rather than the intended "The Gateway, belonging to Discord". Generally speaking "The Discord" (Or any proper noun for that matter) is an emphasis thing, essentially a shorter form of "The one and only Discord".

Reply:
The issue wouldn't be present if you dropped the `'s` though, that makes 'Discord' an adjective, and not a proper noun.